README.md

```diff
@@ -166,6 +166,13 @@ To enable instrumentation libraries manually, see [Manual library instrumentatio
 Documentation on how to manually instrument a Ruby application is available in the 
 [OpenTelemetry official documentation](https://opentelemetry.io/docs/instrumentation/ruby/manual/).
 
+To extend the instrumentation with the OpenTelemetry Instrumentation for Ruby,
+you have to use a compatible API version.
+
+The Splunk Distribution of OpenTelemetry Ruby version 0.1.0 is compatible with:
```

Comment:
For easier maintenance, I suggest changing this line to:

```suggestion
The Splunk Distribution of OpenTelemetry Ruby is compatible with:
```

Consider calling out the supported version in the CHANGELOG. For reference:

- https://github.com/signalfx/splunk-otel-java/blob/main/CHANGELOG.md
- https://github.com/signalfx/splunk-otel-go/blob/main/CHANGELOG.md
- https://github.com/signalfx/splunk-otel-python/blob/main/CHANGELOG.md

Reply:
Done.